platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdminExt.java

```diff
@@ -316,6 +315,18 @@ private void addConfigurationData(Map<String, Object> service, Configuration[] c
                 }
             }
 
+            // If the configuration property is a password, set its value to "password" so that
+            // the real password value will be hidden.
+            List<HashMap<String, Object>> metatypeList =
+                    (List<HashMap<String, Object>>) service.get("metatype");
+            metatypeList.stream()
+                    .filter(metatype -> AttributeDefinition.PASSWORD == (Integer) metatype.get(
+                            "type"))
+                    .forEach(metatype -> {
```

Comment:
Thanks for the suggestion, but I agree this way is cleaner and clearer, so I will keep it that way,